rom/laghos_rom.cpp

```diff
@@ -1285,8 +1285,8 @@ void ROM_Basis::SetupHyperreduction(ParFiniteElementSpace *H1FESpace, ParFiniteE
 
     // This creates sample_pmesh, sp_H1_space, and sp_L2_space only on rank 0.
     CAROM::CreateSampleMesh(*pmesh, H1_space, *H1FESpace, *L2FESpace, *(H1FESpace->FEColl()),
-                     *(L2FESpace->FEColl()), rom_com, sample_dofs_merged,
-                     num_sample_dofs_per_proc_merged, sample_pmesh, sprows, all_sprows, s2sp, st2sp, sp_H1_space, sp_L2_space);
+                            *(L2FESpace->FEColl()), rom_com, sample_dofs_merged,
```

Comment:
I wonder why `make style` did not this fix this already.

Reply:
This was indeed fixed by `make style`. Probably it was just we hadn't do `make style` before.